package/cfg/rke2-cis-1.5/master.yaml

```diff
@@ -0,0 +1,1081 @@
+            set: true
+      remediation: |
+        Run the below command (based on the file location on your system) on the master node.
+        For example,
+        chmod 644 $schedulerconf
+      scored: true
+
+    - id: 1.1.6
+      text: "Ensure that the scheduler pod specification file ownership is set to root:root (Scored)"
+      audit: "/bin/sh -c 'if test -e $schedulerconf; then stat -c %U:%G $schedulerconf; fi'"
+      tests:
+        test_items:
+          - flag: "root:root"
+            compare:
+              op: eq
+              value: "root:root"
+            set: true
+      remediation: |
+        Run the below command (based on the file location on your system) on the master node.
+        For example,
+        chown root:root $schedulerconf
+      scored: true
+
+    - id: 1.1.7
+      text: "Ensure that the etcd pod specification file permissions are set to 644 or more restrictive (Scored)"
+      audit: "/bin/sh -c 'if test -e $etcdconf; then stat -c %a $etcdconf; fi'"
+      tests:
+        test_items:
+          - flag: "644"
+            compare:
+              op: eq
+              value: "644"
+            set: true
+      remediation: |
+        Run the below command (based on the file location on your system) on the master node.
+        For example,
+        chmod 644 $etcdconf
+      scored: true
+
+    - id: 1.1.8
+      text: "Ensure that the etcd pod specification file ownership is set to root:root (Scored)"
+      audit: "/bin/sh -c 'if test -e $etcdconf; then stat -c %U:%G $etcdconf; fi'"
+      tests:
+        test_items:
+          - flag: "root:root"
+            compare:
+              op: eq
+              value: "root:root"
+            set: true
+      remediation: |
+        Run the below command (based on the file location on your system) on the master node.
+        For example,
+        chown root:root $etcdconf
+      scored: true
+
+    - id: 1.1.9
+      text: "Ensure that the Container Network Interface file permissions are set to 644 or more restrictive (Not Scored)"
+      audit: "stat -c %a <path/to/cni/files>"
+      type: "manual"
+      remediation: |
+        Run the below command (based on the file location on your system) on the master node.
+        For example,
+        chmod 644 <path/to/cni/files>
+      scored: false
+
+    - id: 1.1.10
+      text: "Ensure that the Container Network Interface file ownership is set to root:root (Not Scored)"
+      audit: "stat -c %U:%G <path/to/cni/files>"
+      type: "manual"
+      remediation: |
+        Run the below command (based on the file location on your system) on the master node.
+        For example,
+        chown root:root <path/to/cni/files>
+      scored: false
+
+    - id: 1.1.11
+      text: "Ensure that the etcd data directory permissions are set to 700 or more restrictive (Scored)"
+      audit: ps -ef | grep $etcdbin | grep -- --data-dir | sed 's%.*data-dir[= ]\([^ ]*\).*%\1%' | xargs stat -c %a
```

Comment:
This audit doesnt work. `--data-dir` is not passed as a command line argument to the etcd process. its specified in the etcd config file at `/var/lib/rancher/rke2/server/db/etcd/config`

Also, the rke1 test seems to have been simplified to just a stat: https://github.com/rancher/security-scan/blob/master/package/cfg/rke-cis-1.5-hardened/etcd.yaml#L11-L20